system/Test/ReflectionHelper.php

```diff
@@ -97,6 +97,6 @@ public static function setPrivateProperty($obj, $property, $value)
 	public static function getPrivateProperty($obj, $property)
 	{
 		$refProperty = self::getAccessibleRefProperty($obj, $property);
-		return $refProperty->getValue($obj);
+		return is_string($obj) ? $refProperty->getValue() : $refProperty->getValue($obj);
```

Comment:
I'm not sure how this was working previously, but in PHP 8 `ReflectionProperty::getValue()` can *only* take an object parameter. Maybe an undocumented "feature" in 7.x that you could pass a string?